synapse/rest/admin/rooms.py

```diff
@@ -240,6 +240,38 @@ async def on_GET(self, request):
         return 200, response
 
 
+class RoomRestServlet(RestServlet):
+    PATTERNS = admin_patterns("/rooms/(?P<room_id>[^/]+)$")
```

Comment:
Move the `PATTERNS` definition below the docstring so `help` will work properly!